src/components/Editor/SearchBar.css

```diff
@@ -47,7 +47,7 @@
   flex-shrink: 0;
   justify-content: flex-end;
   background-color: var(--theme-toolbar-background);
-  padding: 0 13px;
+  padding: 0 1px 0 13px;
```

Comment:
We should use `padding-inline-start` and `padding-inline-end` here so the design is consistent in RTL.

Reply:
Good catch! I'll update it.